addons/mail/models/res_users_settings_volumes.py

```diff
@@ -14,6 +14,26 @@ class ResUsersSettingsVolumes(models.Model):
     guest_id = fields.Many2one('res.partner', ondelete='cascade', index=True)
     volume = fields.Float(default=0.5, help="Ranges between 0.0 and 1.0, scale depends on the browser implementation")
 
+    def _mail_users_settings_volume_format(self):
```

Comment:
This kind of construct should not be on model ... nothing related to business or records themselves, only some record-to-data convert. Could be great to move towards using controllers in master and stop adding that kind of code on models themselves, to clearly split record management from data conversion.

Anyway, could you avoid using the "_mail_" prefix, as it is used for mail-related stuff ? Chatter, notification, gateway ...  could be great to use "_discuss_" or something similar, as it has nothing to do with mail framework itself.

Reply:
"but until then we just stick to the guideline which is using the module as prefix." Actually that's not a mandatory guideline. Purpose of prefixes is to sort and find stuff. If your guideline leads to noise, your guideline is unnecessary.

For example in mail we use several prefixes: ``_message_`` (main thread API: post, subscribe, ..), ``_mail_`` (tool methods used in mail framework), ``_routing_`` (specific to mail gateway), ``_notify_`` (specific to notification process (mail, sms, ...)). That way it somehow classifies the code (and ease grepping _notify and see the main methods defined for the notification process).

Guidelines is therefore more about classification than about module prefix. Generally module is sufficient, but as mail grows a lot using several prefixes is necessary and helpful I think :) .

Reply:
Oh for the prefix I thought you meant in the PR title, ok in the method name I don't really care what we do.

So just `_users_settings_volume_format`? I don't really see the need for a prefix here. I just want to avoid having the method named too generic (such as just "format") as then we lack tools to properly grep and find it easily.

Reply:
Those could be prefixed by _discuss_ as it helps knowing their usage (preparing data for discuss UI). As you prefer. Just want to keep _mail_something methods as contained as possible ^^ .

Reply:
@ThanhDodeurOdoo please remove the mail prefix from this method then, and also move it at the bottom of the file init and _sql_constraints should be kept close to the top with the fields